pages/items.js

```diff
@@ -80,6 +91,26 @@ export default function ItemsPage() {
             setSorting(newSorting);
           }}
         />
+        <StyledFilter
+          options={[
+            {
+              value: "!isSold",
+              label: "unverkauft",
```

Comment:
just nitpicking: unverkauft sounds like stuff that could not be sold - maybe go with "verfügbar"? 😊

Reply:
I'll keep in eye on it on my QualityOfLife List of fixes 👍🏼